tajo-core/src/test/java/org/apache/tajo/engine/parser/TestSQLAnalyzer.java

```diff
@@ -485,6 +485,18 @@ public void testAlterTableDropPartition3() throws IOException {
   }
 
   @Test
+  public void testAlterTableSetProperty1() throws IOException {
```

Comment:
If possible, could you add more unit test cases such as, text delimiter, compression codec, etc?